src/graphics/graphics_paraboloid.js

```diff
@@ -3,7 +3,7 @@ pc.extend(pc, (function () {
 
     var dpMult = 2.0;
 
-    function paraboloidFromCubemap(device, sourceCubemap, fixSeamsAmount) {
+    function paraboloidFromCubemap(device, sourceCubemap, fixSeamsAmount, dontFlipX) {
```

Comment:
Surely this should be 'flipX'. I don't think this is in the coding standards, but introducing 'don't' into variable names is probably a bad idea. :)